Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -1025,24 +1061,18 @@ final class StatusTableViewController: LoopChartsTableViewController {
             case .iob, .dose, .cob:
                 return max(106, 0.21 * availableSize)
             }
-        case .hud, .status:
+        case .hud, .status, .tempStatus:
             return UITableView.automaticDimension
         }
     }
 
     override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         switch Section(rawValue: indexPath.section)! {
-        case .charts:
```

Comment:
I just reordered these so they match on-screen ordering.  Just thought it was a little neater...